### PR TITLE
STY: wrap lines in raw readme

### PR DESCRIPTION
**Pull request by FirefoxMetzger**
This is a small test PR to check how the rules discussed in https://github.com/pygfx/wgpu-py/issues/301 are working.

The PR reformats the lines in the readme to be (roughly) 80 characters long.

### CODE REVIEW

File: README.md
Comment (Korijn) [inline README.md:17]: Just posting a comment to see what happens...

**Review by Korijn — DISMISSED** [review]

File: README.md
Comment (almarklein) [reply README.md:17]: Resolving makes the button go green. Unresolving disables the merge button again.

**Review by almarklein — DISMISSED** [review]

**Comment by Korijn** [comment]
Did you have to request my review manually or did that happen automatically?

image

**Comment by FirefoxMetzger** [comment]
@Korijn Since you are a code owner the request is created automatically for every new PR that includes code that you own. (Draft PRs are excluded up until they are converted to normal PRs)

**Comment by Korijn** [comment]
Awesome!

**Comment by almarklein** [comment]
Maybe test pushing another commit to check that the approval expires?

**Comment by FirefoxMetzger** [comment]
Looks like the approvals got removed.

**Review by Korijn — APPROVED** [review]

**Comment by FirefoxMetzger** [comment]
Same question as before: Who does the merge once everything is approved? :D

**Comment by Korijn** [comment]
Anyone! You! :)

**Comment by Korijn** [comment]
@FirefoxMetzger please merge or close this PR. :)

**Pull request merged**
